scripts/docs.js

```diff
@@ -20,8 +22,14 @@ function validatePublicFiles(definitionFiles) {
   }
 }
 
+function getNodeModulesInterfacePath(componentName) {
+  return cwd + "/node_modules/@cloudscape-design/components/" + componentName + "/interfaces.d.ts";
+}
+
 function componentDocs() {
-  const definitions = documentComponents(path.resolve("tsconfig.json"), "src/*/index.tsx");
+  const importedCoreComponents = ["icon"];
```

Comment:
Since this one is a constant, we could as well move it out of the function's body to the top of the file.